### PR TITLE
Tokenized the dark and light themes

### PR DESCRIPTION
**Pull request by brentschneider**
# Proposed Changes

Describe the big picture of your changes here.

## Checklist

- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation

- [x] My changes generate no new warnings
- [ ] Logging has been added where applicable (Error handling, user-driven events, API requests/responses)

- [x] I have added tests that prove my fix is effective or that my feature works (if applicable)
- [ ] Tests meet the test coverage threshold

## Reviewer Checklist

- [ ] Is the code easy to understand?
- [ ] Are the changes well documented?

- [ ] Can you think of any use case in which the code does not behave as intended?
- [ ] Can you think of any inputs or external events that could break the code?

- [ ] Should any logging or debugging information be added or removed?

## Further Comments

I tokenized the dark and light themes to use color tokens defined in the color types file.
In Figma, I ran a comparative Q/A of our colors then extended our color types file to include all base colors found on VADS at: design.va.gov/design/color-palette

### CODE REVIEW

**Review by rianfowler — APPROVED** [review]

**Review by KaemonIsland — APPROVED** [review]

File: packages/app/src/themes/dark_theme.tsx [hunk 1]
```diff
@@ -1,5 +1,5 @@
 import { createTheme, SimpleThemeOptions } from '@backstage/theme';
-import { gold, common, myYellow as yellow } from "./colorTypes";
+import { blue, gray, green, gold, red, common } from "./colorTypes";
```
Comment (rianfowler) [inline packages/app/src/themes/dark_theme.tsx:2]: This import is failing an unused variable check. 

You can run this check locally with `npx tsc` and you can see the output from this run [here](https://github.com/department-of-veterans-affairs/lighthouse-backstage/pull/168/checks?check_run_id=3749804674). 
Screen Shot 2021-09-30 at 4 01 40 PM

Comment (rianfowler) [inline packages/app/src/themes/dark_theme.tsx:2]:
```suggestion
import { blue, /* gray, green, gold, red, common */ } from "./colorTypes";
```

If these don't need to be used we can delete or comment them out.

Comment (rianfowler) [inline packages/app/src/themes/dark_theme.tsx:2]:
```suggestion
 /* tslint:disable */
import { blue, gray, green, gold, red, common } from "./colorTypes";
 /* tslint:enable */
```

If they are needed we can disable the check for this one line like this.

**Review by rianfowler — APPROVED** [review]

File: packages/app/src/themes/dark_theme.tsx (hunk 1 above)
Comment (brentschneider) [reply packages/app/src/themes/dark_theme.tsx:2]: What would be the preferred approach? My thought was to set the stage for using the color variables but I don't need them just yet. 

In the two methods you posted I'm leaning towards the first, commenting them out vs setting a flag to ignore.

Comment (rianfowler) [reply packages/app/src/themes/dark_theme.tsx:2]: That works for me.

**Pull request merged**